app/models/account_user.rb

```diff
@@ -44,17 +44,23 @@ def self.active
   # Provides an +Array+ of roles that can be assigned
   # to a user. Optionally filters the set by the given
   # arguments
-  # [_user_]
+  # [_granting_user_]
   # The user selecting a role to be applied to
   # another user; the grantor
   # [_facility_]
   # The facility under which the selected role is
   # granted by +user+
-  def self.selectable_user_roles(user=nil, facility=nil)
-    user_roles.reject { |r| r == ACCOUNT_OWNER if user.nil? || facility.nil? || !user.manager_of?(facility) }
+  def self.selectable_user_roles(granting_user = nil, facility = nil)
+    case
+    when granting_user.blank? || facility.blank?
```

Comment:
Gotcha. I'm not convinced it's worth the effort to clean it up right now, but if you think it'll be easy, go ahead.

Otherwise, feel free to merge. :+1: